src/substrate/extrinsic.rs

```diff
@@ -18,6 +18,8 @@ pub struct ExtrinsicsQuery {
     pub pallet: Option<String>,
     /// Optional call name filter
     pub call: Option<String>,
+    /// Optional flag to return only signed or unsigned extrinsics
+    pub is_signed: Option<bool>,
```

Comment:
This overlaps a bit with `signer` below, I would say the following is invalid:

```rust
let query = ExtrinsicQuery {
    is_signed = Some(false),
    signer = Some("0x..."),
    ..
}
```

Probably not a big deal, but I thought I'd mention it.